frontend/model/notifications/vuexModuleKeys.js

```diff
@@ -0,0 +1,14 @@
+
+/* Action keys */
+
+export const LOGIN = 'login'
+export const LOGOUT = 'logout'
+
+/* Mutation keys */
+
+export const ADD_NOTIFICATION = 'addNotification'
+export const MARK_ALL_NOTIFICATIONS_AS_READ = 'markAllNotificationsAsRead'
+export const MARK_NOTIFICATION_AS_READ = 'markNotificationAsRead'
+export const REMOVE_ALL_NOTIFICATIONS = 'removeAllNotifications'
+export const REMOVE_NOTIFICATION = 'removeNotification'
+export const SET_NOTIFICATIONS = 'setNotifications'
```

Comment:
Yes, this module is not namespaced, so that Vuex will effectively mix its actions and mutations with main ones defined in the root module `state.js`.
So for example upon login, the `login` action handler from `notifications/vuexModule.js` will be automatically called alongside its eponym from `state.js`. This works because they have the same name.

Reply:
However exporting them might have been a mistake. The intent was to allow usage of LOGIN and LOGOUT constants in `vuexModule.js`, but whether to export them would rather be the responsability of `state.js`.

Reply:
So my concern is the inconsistency and potential DRY issues.

DRY is designed to prevent multiple copies of the same code from appearing in the codebase, which over time leads to chaos, disorganisation, and an unmaintainable codebase. It's entropy that has to always be fought against.

So if in one part of the codebase we're using `'login'` as defined in `state.js`, and in another part of the codebase we're using `LOGIN`, that's the kind of thing to avoid.

I am definitely OK with using constants, but if you introduce them, then you have to modify all of the rest of the code to do the same thing.

Reply:
Agreed, randomly using either `LOGIN` or `'login'` would be inconsistent. Therefore `LOGIN` will no longer be exported for now, leaving only `login`. We can still think later about exporting constants from `state.js` and refactor as necessary to use them.